atlas/commands/bootstrap/bootstrap.go

```diff
@@ -162,7 +162,9 @@ func (app Application) initializeDirectories() error {
 // getDirectories returns a list of all project folders
 func (app Application) getDirectories() []string {
 	dirnames := []string{
+		"cmd",
```

Comment:
why are these required now? I'm assuming it worked before without them

Reply:
It worked without these, but `goimports` would skip over `cmd` and `pkg` because they're not explicitly listed as directories in the project. Make sense? It would just run `goimports` on the child directories of `cmd`.